arcsi/api/show.py

```diff
@@ -108,7 +108,7 @@ def view_show(id):
 
 
 @arcsi.route("/show/<string:slug>/archive", methods=["GET"])
-def view_archive(slug):
+def view_show_archive(slug):
```

Comment:
Good naming schema thanks!

Reply:
Thanks, I tried to be as clear as I can. What do you think about rename this route to `"/<string:show_slug>/archive"` in order to abandon the redundant `/show` prefix?

Reply:
There's a discussion in chat that I will join and we can discuss there